### PR TITLE
updating docs and gif

### LINKED ISSUE #47: update documentation to reflect v0.6 changes
The documentation needs to updated to show the changes made to the stats command in #46. This would include changing the README text and the gif attached.

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Fixes #47

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,4 +27,29 @@ Custom Information - if you're copying this template for the first time you can
...
 
-- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
\ No newline at end of file
+- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
+
+## PR Workflow
+
+1. Clone the repository by running 
+```
+https://github.com/kubernetes-sigs/depstat.git
```
Comment (nikhita) [inline CONTRIBUTING.md]:
```suggestion
git clone https://github.com/kubernetes-sigs/depstat.git
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,4 +27,29 @@ Custom Information - if you're copying this template for the first time you can
 
 ## Contact Information
 
-- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
\ No newline at end of file
+- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
+
+## PR Workflow
```
Comment (nikhita) [inline CONTRIBUTING.md:32]: optional - you can also add a link to https://www.kubernetes.dev/docs/guide/github-workflow/ for more info on the GitHub workflow

File: README.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+To install depstat you can run
+
+```
+go get github.com/kubernetes-sigs/depstat@latest
+```
 
 ## Usage
-`depstat` can be used as a standalone command-line application. It is also to be used in the [Kubernetes](https://github.com/kubernetes/kubernetes) CI pipeline. For more info on the latter see [#100792](https://github.com/kubernetes/kubernetes/pull/100792).
+`depstat` can be used as a standalone command-line application. You can navigate to your go modules enabled project and use `depstat` to produce metrics about your project. Another common way to run `depstat` is in the CI pipeline of your project. This would help you analyze the dependency changes which come with PRs. You can look at how this is done for the [kubernetes/kuberenets](https://github.com/kubernetes/kubernetes) repo using [prow](https://github.com/kubernetes/test-infra/tree/master/prow) [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml).
```
Comment (nikhita) [inline README.md]: nit: would be good to break this down into multiple lines foe easier review.

> You can navigate to your go modules enabled project and use `depstat` to produce metrics about your project.

nit: maybe rephrase as `You can use depstat to produce metrics about the dependencies of your go modules enabled project.`?

> You can look at how this is done for the [kubernetes/kuberenets](https://github.com/kubernetes/kubernetes)

nit: typo in kuberenets :)

File: README.md [hunk 4]
```diff
@@ -41,29 +45,40 @@ An example of a cycle in project dependenies is:
...
+3. Total Dependencies: Total number of dependencies of the [main module(s)](#main-module).
 
-3. Transitive Dependencies: Total number of transitive dependencies.
+4. Max Depth of Dependencies: Length of the longest chain starting from the first [main module](#main-module); defaults to length from the first module encountered in "go mod graph" output.
 
 - The `--json` flag gives this output in a JSON format.
 - `--verbose` mode will help provide you with the list of all the dependencies and will also print the longest dependency chain.
 
+#### main module
+By default, the first module encountered in "go mod graph" output is treated as the main module by `depstat`. Depstat uses this main module to determine the direct and transitive dependencies. This behavior can be changed by specifying the main module manually using the `--mainModules` flag with the stats command. The flag takes a list of modules names, for example:
```
Comment (nikhita) [inline README.md:78]: tiny nit: The `--mainModules` flag is mentioned here but the `-m` flag is used below. Maybe mention the `-m` flag in the text description as well?

Comment (RinkiyaKeDad) [reply README.md:78]: Replaced it in the example to avoid more text :)

**Comment by RinkiyaKeDad** [comment]
Thanks for reviewing! Made the suggested edits, PTAL :)

**Comment by nikhita** [comment]
/lgtm

/hold
please feel free to cancel the hold when ready to merge/at your discretion
